src/content/en/tools/workbox/modules/workbox-webpack-plugin.md

```diff
@@ -0,0 +1,82 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: The module guide for workbox-webpack-plugin.
+
+{# wf_updated_on: 2017-12-15 #}
+{# wf_published_on: 2017-12-15 #}
+{# wf_blink_components: Blink>ServiceWorker #}
+
+# Workbox webpack Plugin  {: .page-title }
+
+## What's the Workbox webpack Plugin?
+
+This is a plugin for the [webpack](https://webpack.js.org/) which generates a list of URLs to
+[precache](workbox-precaching) (known as a "precache manifest") based on the assets in a webpack
+compilation.
+
+## Considerations
+
+There are a few decisions to make which will determine the right configuration for your use case.
+
+### Should the plugin create your service worker file?
+
+If you leave out the `swSrc` configuration option, the plugin will create a service worker file for
+you and automatically add it to the webpack asset pipeline. This is referred to as `generateSW`
+mode.
+
+{% include "web/tools/workbox/_shared/when-to-use-generate-sw.md" %}
+
+If you set `swSrc` to the path of an existing service worker file (which does not have to be
+otherwise configured as a webpack asset) then the plugin will generate a list of URLs to precache
+and add a reference to that precache manifest, along with a reference to the Workbox runtime code,
+to your file. It will otherwise leave the file as-is. This is referred to as `injectManifest` mode.
+
+{% include "web/tools/workbox/_shared/when-to-use-inject-manifest.md" %}
+
+### Do you need to cache additional, non-webpack assets?
+
+By default, the plugin will generate a precache manifest that contains URLs for all of the assets
+that come from the current webpack compilation. Any assets that webpack doesn't "know" about will
+not be picked up.
+
```

Comment:
So if you'd copied something into the same output location as that which webpack uses then this would **not** be included.  i.e. You would need to use `globDirectory` and `globPatterns` in this case.  Not otherwise.

Reply:
Right.